wdl-engine/Cargo.toml

```diff
@@ -41,6 +41,9 @@ reqwest = { workspace = true }
 reqwest-middleware = { workspace = true }
 dirs = { workspace =  true }
 
+[target.'cfg(target_os = "linux")'.dependencies]
```

Comment:
@peterhuene is this right? I think in other places we have conditional compilation we do `windows` and `not(windows)`. Is this issue not present on MacOS?

Reply:
Yes, it is also a macOS issue as well.

Reply:
using `libc` now directly 👍